src/SFA.DAS.EmployerIncentives.Api/NLog.local.config

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
+  <extensions>
+    <add assembly="SFA.DAS.NLog.Targets.Redis" />
+    <add assembly="Microsoft.ApplicationInsights.NLogTarget" />
+  </extensions>
+  <variable name="appName" value="das-employer-incentives-api"/>
+  <variable name="simplelayout" value="${longdate} [${uppercase:${level}}] [${logger}] - ${message} ${onexception:${exception:format=tostring}}"/>
+  <targets>
+    <target name="Console" layout="${simplelayout}" xsi:type="Console"/>
```

Comment:
Please add a target to output to the VS debug window i.e.

 <target name="Debugger" layout="${simplelayout}" xsi:type="Debugger"/>
 
 and a logger below
 
 <logger name="SFA.DAS.*" minlevel="Debug" writeTo="Debugger" />